classes/Cart.php

```diff
@@ -3099,116 +3069,6 @@ public function carrierIsSelected($id_carrier, $id_address)
         return true;
     }
 
-    /**
-     * Get all deliveries options available for the current cart formatted like Carriers::getCarriersForOrder
-     * This method was wrote for retrocompatibility with 1.4 theme
-     * New theme need to use Cart::getDeliveryOptionList() to generate carriers option in the checkout process.
-     *
-     * @since 1.5.0
-     * @deprecated 1.7.0
-     *
-     * @param Country $default_country Default Country
-     * @param bool $flush Force flushing cache
-     *
-     * @return array
-     */
-    public function simulateCarriersOutput(Country $default_country = null, $flush = false)
-    {
-        $delivery_option_list = $this->getDeliveryOptionList($default_country, $flush);
-
-        // This method cannot work if there is multiple address delivery
-        if (count($delivery_option_list) > 1 || empty($delivery_option_list)) {
-            return [];
-        }
-
-        $carriers = [];
-        foreach (reset($delivery_option_list) as $key => $option) {
-            $price = $option['total_price_with_tax'];
-            $price_tax_exc = $option['total_price_without_tax'];
-            $name = $img = $delay = '';
-
-            if ($option['unique_carrier']) {
-                $carrier = reset($option['carrier_list']);
-                if (isset($carrier['instance'])) {
-                    $name = $carrier['instance']->name;
-                    $delay = $carrier['instance']->delay;
-                    $delay = isset($delay[Context::getContext()->language->id]) ?
-                        $delay[Context::getContext()->language->id] : $delay[(int) Configuration::get('PS_LANG_DEFAULT')];
-                }
-                if (isset($carrier['logo'])) {
-                    $img = $carrier['logo'];
-                }
-            } else {
-                $nameList = [];
-                foreach ($option['carrier_list'] as $carrier) {
-                    $nameList[] = $carrier['instance']->name;
-                }
-                $name = implode(' -', $nameList);
-                $img = ''; // No images if multiple carriers
-                $delay = '';
-            }
-            $carriers[] = [
-                'name' => $name,
-                'img' => $img,
-                'delay' => $delay,
-                'price' => $price,
-                'price_tax_exc' => $price_tax_exc,
-                'id_carrier' => Cart::intifier($key), // Need to translate to an integer for retrocompatibility reason, in 1.4 template we used intval
-                'is_module' => false,
-            ];
-        }
-
-        return $carriers;
-    }
-
-    /**
-     * Simulate output of selected Carrier.
-     *
-     * @param bool $use_cache Use cache
-     *
-     * @return int Intified Cart output
-     */
-    public function simulateCarrierSelectedOutput($use_cache = true)
```

Comment:
2 functions Cart::containsProduct and Cart::simulateCarriersOutput are clearly deprecated but I'm a bit worry about 3 others
Cart::simulateCarrierSelectedOutput
Cart::intifier
Cart::desintifier

Reply:
If we want to delete them, we have to deprecate them first.

Reply:
Done, deprecated only